### DISCUSSION TITLE
第2节课作业(8班)

### DISCUSSION THREAD

**Discussion by Hongru0306** [InternLM/Tutorial#112] [作业]
请在下方提交作业链接,格式为
-->报名序号+连接:
例如:213+https://kvudif1helh.feishu.cn/docx/...

**Comment by NYYY0421** [comment]
501+https://github.com/NYYY0421/homework1/blob/main/homework1.md

**Comment by zhaoxin111** [comment]
445+https://app.yinxiang.com/fx/0344b221-d697-4eee-93db-fcc903c91e72

**Comment by Alias-z** [comment]
388+https://github.com/Alias-z/InternLM-Camp-2024/blob/main/Assignment%201/README.md

**Reply by Alias-z** [reply]
更新一下链接 文件夹改名了 https://github.com/Alias-z/InternLM-Camp-2024/tree/main/Assignment%201%20InternLM-Chat-7B

**Comment by ZPfree** [comment]
605+https://github.com/ZPfree/homework/blob/main/1.md

**Comment by BJUT-WangZhengTao** [comment]
371+https://blog.csdn.net/m0_54169072/article/details/135432260

**Comment by superkong001** [comment]
545+https://github.com/superkong001/InternLM_Learning/blob/main/homework1.md

**Comment by baijiesong** [comment]
509+https://github.com/baijiesong/InternLM_Learning/blob/main/%E7%AC%AC%E4%BA%8C%E8%8A%82%E4%BD%9C%E4%B8%9A.md

**Comment by 1212jake** [comment]
553+https://github.com/1212jake/-1-8-/blob/main/2readme.md

**Comment by jamiechoi1995** [comment]
554+https://blog.csdn.net/sinat_26253653/article/details/135437864

**Comment by Vincent-luo** [comment]
566+https://www.yuque.com/vincentluo-wbw4o/xpmg2z/bdey08o7eok0pat0?singleDoc# 《作业2》

**Comment by JieGenius** [comment]
555+https://github.com/JieGenius/InternLMLearning/blob/main/lesson2/assigniment.md

**Comment by L-369-maha** [comment]
521+https://github.com/L-369-maha/InternLM/blob/main/Two/Task.md

**Comment by Zeno673** [comment]
567+https://github.com/Zeno673/InternLMLearning/blob/main/homework/homework_2.md

**Comment by Yujin320** [comment]
587+https://github.com/Yujin320/InternLM-hyx-Learning/blob/main/homework02.md

**Comment by Cyydz** [comment]
527+https://www.wolai.com/3oPwED5Ky7L7a9Cfa2vCA6

**Comment by jzh999** [comment]
361+https://www.yuque.com/bright-p9un8/yum3v8/dpywvg2lk66lf6sh?singleDoc# 《第二节、轻松玩转书生·浦语大模型趣味 Demo（作业）》

**Comment by hzsun1995** [comment]
446+https://github.com/hzsun1995/internlm-course/blob/main/course2.md

**Comment by carlinl** [comment]
596 + https://pt35a5ibej.feishu.cn/docx/Ngdidti3Zox6DOxknrqcWOdfnQe?from=from_copylink

**Comment by Fautines** [comment]
579+https://www.yuque.com/shijiuchennianhua-avnma/pn9l3t/gkaa8btg8rnslh7h?singleDoc# 《day2作业》

**Comment by sunxiaobei** [comment]
585+https://github.com/sunxiaobei/InternLM_tutorial/blob/main/notes/HWDay02.md

**Comment by Doublefei-Giser** [comment]
409+https://blog.51cto.com/u_16507376/9165476
